app/views/messages/_editor.haml

```diff
@@ -2,31 +2,26 @@
 
 = form_for @message, html: { id: 'message_form' } do |f|
   = hidden_field_tag :parent_id, @message.parent_id
-  %table
-    %thead
-      %tr
-        %th.table-title{colspan: 2}
-          - if @message.thread_id.present?
-            Reply to Thread
-          - else
-            Compose New Message
-    %tbody
-      %tr
-        %th.sub.padding-10.width-150= f.label :recipient_id
-        %td.even.padding-10
-          - if @message.parent
-            = user_link(@message.recipient) if @message.recipient
-          - else
-            = f.select :recipient_id, @select_items, prompt: true
-      - unless @message.thread_id.present?
-        %tr
-          %th.sub.padding-10= f.label :subject
-          %td.odd.padding-10= f.text_field :subject, placeholder: 'Subject'
-      %tr#message-row
-        %th.sub.padding-10.vtop= f.label :message
-        %td.even.padding-10= f.text_area :message
-    %tfoot
```

Comment:
![image](https://github.com/user-attachments/assets/9763796b-a8bd-479b-98d3-08b0aa808876)
![image](https://github.com/user-attachments/assets/4e7a037b-e0e9-4058-8743-2afebad3e142)

I think this is the same thing as earlier, the first column is too wide? It's not as big a deal here, but.